### PR TITLE
Dataset command

### PR DESCRIPTION
**Pull request by metaodi**

### CODE REVIEW

File: ckanext/geocat/commands.py [hunk 1]
```diff
@@ -21,37 +24,75 @@ def command(self):
...
 
-    def cqlCmd(self, query=None):
+    def cqlCmd(self, query=None, csw_url=None):
         if (query is None):
             print "Argument 'query' must be set"
             self.helpCmd()
             sys.exit(1)
-        csw = md.CswHelper('http://www.geocat.ch/geonetwork/srv/eng/csw')
+        if csw_url is None:
+            csw_url = 'http://www.geocat.ch/geonetwork/srv/eng/csw'
```
Comment (stefina) [inline ckanext/geocat/commands.py]: The same URL-Value is used several times. It is just a default, but maybe it is easier to read, if it is set once as a constant and re-used as a `default_url`.
Same with `default_cql` maybe?

File: README.md [hunk 2]
```diff
@@ -42,6 +42,60 @@ To configure the harvester you have several harvester config options (in the har
...
 
 
+## CLI Commands
+
+This extension provides a number of CLI commands to query/debug the results of the CSW server.
+
+
+### `search`
+
+To run a query an arbitrary query against the defined CSW server use the `search` command:
```
Comment (stefina) [inline README.md]: Something in this sentence sounds off, I mean duplicated :)

File: README.md [hunk 3]
```diff
@@ -42,6 +42,60 @@ To configure the harvester you have several harvester config options (in the har
...
+
+```
+paster geocat search birds
+```
+
+This command takes an optional second parameter to specifiy the CSW url (defaults to `http://www.geocat.ch/geonetwork/srv/eng/csw`)
+
+### `cql`
+
+To run a query an arbitrary CQL query against the defined CSW server use the `cql` command:
```
Comment (stefina) [inline README.md]: Something in this sentence sounds off, I mean duplicated :)

File: README.md [hunk 4]
```diff
@@ -42,6 +42,60 @@ To configure the harvester you have several harvester config options (in the har
...
+```
+paster geocat cql "csw:AnyText like '%birds%'"
+paster geocat cql "keyword = 'opendata.swiss'" https://www.geocat.ch/geonetwork/srv/eng/csw-ZH
+```
+
+This command takes an optional second parameter to specifiy the CSW url (defaults to `http://www.geocat.ch/geonetwork/srv/eng/csw`)
+
+### `list`
+
+To list all IDs from the defined CSW server use the `list` command:
```
Comment (stefina) [inline README.md]: Does calling it `server-endpoint` make it a little more clear?

File: README.md (hunk 2 above)
Comment (metaodi) [reply README.md]: Fixed.

File: README.md (hunk 3 above)
Comment (metaodi) [reply README.md]: Fixed.

**Pull request merged**